js/site.js

```diff
@@ -20,12 +40,17 @@ class Site{
     getScore(){
         this.score = this.scoreFunction();
         return this.score;
-    }
+    };
 
-    getTrackers(){ return this.trackers };
+    _checkGlobalWhitelist(domain){
```

Comment:
I see. `checkGlobalWhitelist` is not a very good name then. The idea was that `checkGlobalWhitelist` only runs when the object is created to look up any stored whitelist values in settings. After you have an object you would change its whitelist status using `setWhitelisted`.